website/redirects.json

```diff
@@ -159,7 +159,7 @@
 {"source": "ingestion/command-line-hadoop-indexer.html", "target": "hadoop.html#cli"}
 {"source": "ingestion/compaction.html", "target": "data-management.html#compact"}
 {"source": "ingestion/delete-data.html", "target": "data-management.html#delete"}
-{"source": "ingestion/firehose.html", "target": "native-batch.html#firehoses"}
+{"source": "ingestion/firehose.html", "target": "native-batch.html#firehoses-deprecated"}
```

Comment:
How did you find these broken links? I wonder if we could add a way to detect these dead links automatically.

Reply:
obviously, no need to do anything in this PR - I was just curious...

Reply:
We actually have this, both for normal links and [these redirects](https://github.com/apache/druid/blob/master/website/script/do-redirects.js#L78), and it is how I found them, not sure why it wasn't failing the build though:

```
On line 35 could not find anchor 'firehoses' in 'ingestion/native-batch.html'
On line 50 target development/integrating-druid-with-other-technologies.html does not exist
On line 76 target operations/recommendations.html does not exist
On line 162 could not find anchor 'firehoses' in 'ingestion/native-batch.html'
There are 4 issues with the redirects
```